### ISSUE TITLE
Make Cornelius pass the antperson test

### ISSUE THREAD

**Issue by joostdecock** [freesewing/freesewing#1317]
Drafting on a non-human scale has been added to the automated unit tests.
Cornelius is one of the patterns that fails these tests.

We should fix that.

**Comment by woutervdub** [comment]
@joostdecock, how can I run these tests, or see the results?

**Comment by joostdecock** [comment]
If you have pulled in the latest changes, you can run `yarn test` or `npm run test` to run the tests (in the cornelius folder).
I plan to add them to the dev environment but they're not there yet.

You can edit the test file in cornelius and pass and extra `true` parameter to this method to log the errors:

https://github.com/freesewing/freesewing/blob/d13d0118e980d56796433a0ba177a882536536fd/packages/cornelius/tests/shared.test.js#L23-L30

Basically change line 29 to:
```js
  patterns, true
```

**Comment by joostdecock** [comment]
After some fixed to the unit tests, Cornelius passes.
